TitanRobotics2024/src/main/java/frc/robot/Auto/AutoMissionExecutor.java

```diff
@@ -21,11 +21,13 @@ public void run(){
         });
     }
 
```

Comment:
make sure the starting curly brackets are on the next line

Reply:
#4